### PR TITLE
Added the weather api based on current location

### PR DESCRIPTION
**Pull request by Faro0q**
Added weather based on your location
image

Currently, it doesnt matter if the user has not typed in their location because we are getting the weather based on the location:
image
image

The reason its there is to just populate the field it self when ever they type in their location:
image
image

The API is getting retrieved from the server. Please create a .env file in the root of the directory and add in your key there
image
`REACT_APP_WEATHER_API_KEY = 'xx'`

TODO: 
Need to add a submit button that will control the flow and display the weather once a location is entered. Currently, its being displayed when the user allows location sharing.

Might implement a google cloud maps api instead to avoid the hassle of location text box. 
Also, the location doesnt matter and can just remove it all together where the weather will look like this:
image

### CODE REVIEW

File: client/src/Common.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const routes = {
 	LOGIN: "/account/login",
 	TWEET: "/tweet",
 	TWEETS_FROM_USER: "/tweets/user/",
-	GET_ALL_TWEETS: "/tweets/all"
+	GET_ALL_TWEETS: "/tweets/all",
+	GET_API_KEY: "/weather_api"
```
Comment (wcygan) [inline client/src/Common.js:7]: `/weather-api`?

File: server/server.js [hunk 2]
```diff
@@ -147,5 +150,17 @@ app.get(routes.GET_ALL_TWEETS, async (req, res, next) => {
 	})(req, res, next);
 });
 
+app.get(routes.GET_API_KEY, async (err, res) => {
+	if (res) {
+		res.status(StatusCodes.OK).json(process.env.REACT_APP_WEATHER_API_KEY);
+	}
+});
+
+app.get("/protected-hello", passport.authenticate("jwt", { session: false }),
```
Comment (wcygan) [inline server/server.js:159]: why is this being added back?

**Review by wcygan — CHANGES_REQUESTED** [review]

File: client/src/Common.js (hunk 1 above)
Comment (Faro0q) [reply client/src/Common.js:7]: Send over api key

File: server/server.js (hunk 2 above)
Comment (Faro0q) [reply server/server.js]: 🥇

**Comment by wcygan** [comment]
🤠 thank u

**Review by wcygan — APPROVED** [review]

**Pull request merged**